code/game/machinery/teleporter.dm

```diff
@@ -141,10 +142,48 @@
 	for(var/obj/effect/landmark/minimap_poi/poi in SSholomap.pois)
 		if(poi.z in SSodyssey.scenario_zlevels)
 			possible_pois += poi
+	if(isemptylist(possible_pois))
```

Comment:
This isn't a lazy list, you defined it above, so it shouldn't use this macro (that is just `!LAZYLEN()`)

Reply:
why not
`LAZYLEN` is just `length(L)`

what am I supposed to use to check if a list is empty if not `isemptylist()`

if this is ONLY for lazy lists then it should be documented there (it's not)

Reply:
You should use lazylist macros to interact with lazylists, and not use them when not interacting with lazylists, hence this would be `!length(list)` if it isn't a lazylist (and in fact this macro would probably disappear at some point, because it's just an useless `!LAZYLEN()`, and doesn't even do what it says because it also applies if the list is null, not just empty, but that's another topic for another day)

Reply:
replaced with !length